Content.Goobstation.Shared/Dash/DashActionComponent.cs

```diff
@@ -45,4 +46,7 @@ public sealed partial class DashActionEvent : WorldTargetActionEvent
     /// </summary>
     [DataField]
     public ProtoId<EmotePrototype>? Emote = "Flip";
+
+    [DataField]
+    public string? Speech = "65!";
```

Comment:
the price for the approve
```suggestion
    public string? Speech;

Reply:
Wait